.github/workflows/vulnerability-scanning-on-repo.yml

```diff
@@ -15,6 +15,10 @@ on:
         required: false
         default: .
         type: string
+      TRIVY_IGNORES:
+        required: false
+        type: string
+        description: comma-separated list of relative paths to .trivyignore files
```

Comment:
Discussed elsewhere but as @tatelicent has stated we don't want to allow any High or Critical vulnerabilities, it doesn't particularly matter if we have analysed them and don't believe they impact us that won't fly with the customer.  Using ignore files **SHOULD** be avoided in favour of actually doing whatever work is needed to actually upgrade dependencies to versions that are not impacted by vulnerabilities

Reply:
Yep, reversed in my WIP branch 👍

Reply:
removed in this [squashed merge](https://github.com/telicent-oss/shared-workflows/pull/45/commits/b68c6ab0b046d2d45d95e35a0c87d71ba5a87d1d#diff-09c3260e3ea2f3dbef81847165c2f334b467b9f634e54b41801487ce570f9896L83-L85)

(isolated in [Pre-squash](https://github.com/telicent-oss/shared-workflows/pull/61/commits/f1faa2074fb4f3ffec29d02e7b992e02886f06d6))

Reply:
@rvesse could you re-review in light of the above